### PR TITLE
chore: release 0.14.0 :)

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

**Comment by machi1990** [comment]
Should we do something about the website?

File: website/versioned_docs/version-0.14.0/.old/commands.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+title: CLI commands
+---
+
+## Commands
+
+CLI offers help prompt. 
+Execute `graphback` in your shell for more information
+
+### Config 
+
```
Comment (machi1990) [inline website/versioned_docs/version-0.14.0/.old/commands.md:12]: I guess it is time to remove the [`/.old`](https://github.com/aerogear/graphback/blob/1231a166c80e3254ff19a0deb4d4040a2cdc1151/website/versioned_docs/version-0.14.0/.old) folder.

Comment (machi1990) [reply website/versioned_docs/version-0.14.0/.old/commands.md:12]: And the [parent source](https://github.com/aerogear/graphback/tree/1231a166c80e3254ff19a0deb4d4040a2cdc1151/docs/.old) folder too

Comment (wtrocki) [reply website/versioned_docs/version-0.14.0/.old/commands.md:12]: Ouch :D

**Comment by wtrocki** [comment]
Done

**Comment by wtrocki** [comment]
Website is live :D

**Pull request merged**